tests/causal_refuters/test_dummy_outcome_refuter.py

```diff
@@ -2,10 +2,24 @@
 import numpy as np
 from .base import TestRefuter
 
+def linear_regression(df):

```

Comment:
It is a good idea to not use common names like linear_regression, since they can be confused with actual linear_model or LinearRegression from sklearn or other packages. 
How about linear_outcome_model, or since this model is quite simple, "simple_linear_outcome_model"?

Reply:
I renamed the function to simple_linear_outcome_model